Druid/DRCogitCodeGenerator.class.st

```diff
@@ -1277,6 +1291,20 @@ DRCogitCodeGenerator >> visitSubstract: aDRCogitSubstract [
 
 ]
 
+{ #category : 'visiting' }
+DRCogitCodeGenerator >> visitTestAnd: aDRTestAnd inBranch: aDRBranchIfCondition [ 
+	
+	| comparands |
+	comparands := aDRBranchIfCondition operands.
+	aDRBranchIfCondition operands second isConstant ifTrue: [ 
+		comparands := comparands reversed ].
+	self
+		twoOperandCogitRTL: #And
+		operands: comparands
+		instruction: aDRBranchIfCondition.
+	^ aDRBranchIfCondition.
+]
```

Comment:
Can we reuse the logic of `visitTestMask: aDRTestMask inBranch: aDRBranchIfCondition`?

Or maybe we already have something, for example: `threeOperandCogitRTL: aMnemonic instruction: instruction`